stages/org.osbuild.pki.update-ca-trust

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
+import sys
+
+import osbuild.api
+from osbuild.util.chroot import Chroot
+
+if __name__ == '__main__':
+    args = osbuild.api.arguments()
+    with Chroot(args["tree"]) as chroot:
+        ret = chroot.run(["/usr/bin/update-ca-trust", "extract"])
```

Comment:
So after refactoring to `chroot` linter revealed that it does not return a value. So I am going ahead and amending a small change adding the missing `return` statement. I think it might be useful going forward what you think?